tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -337,6 +338,7 @@ namespace EnergyPlus {
 		HVACControllers::clear_state();
 		HVACDXHeatPumpSystem::clear_state();
 		HVACDXSystem::clear_state();
+		HVACFan::clearHVACFanObjects();
```

Comment:
Well it works a little differently than clear_state does in other places, but yes it could be called the same.